### ISSUE TITLE
Parameters not working

### ISSUE THREAD

**Issue by EricNetsch** [redcap3000/meteor-wordpress-json-api#4]
This plugin works well for the most part. Seems like all parameters are being stripped out during the api GET. The URL returns the proper content when viewed in the browser but doesn't work when used in meteor. 

 Please take a look at this and let me know if there is an update or fix.
Thanks,

**Comment by redcap3000** [comment]
I'm not sure what you mean by 'proper content'. This is not intended to mirror the content from your wordpress site by letting you access it using similar URLs. This package is designed to store data from your wordpress site to your mongo database. Then it is up to you how to display that information, there isn't a built-in url structure to use to access your data within meteor.

**Comment by EricNetsch** [comment]
Hi sorry for the delay. By 'proper content' I meant the JSON response which is viewable from a web browser by visiting the endpoint URL

**Comment by redcap3000** [comment]
Yeah I'll look into it.. haven't touched wordpress in years.. but my first inclination is that your webserver hosting the wordpress plugin/installation isn't properly configured and may be returning an XML/RSS or poorly formed JSON response.

Next it might be that your server hosting the wordpress site is doing some interesting things to redirect with parameters (for security purposes.. I'm assuming) Which may stem from improperly configured SSL certificates / CORS stuff.
